terra-key

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Try to keep private key file only on local machine. Don't add it into the repo.

Reply:
undestood